src/components/Quiz.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import Answers from './Answers';
+import { publish } from './Event';
+import QuestionAnswer from './QuestionAnswer';
+import Score from './Score';
+import TimerContainer from './TimerContainer';
+
+function Quiz(props) {
+  const { quiz } = props;
+  const [questionNumber, setQuestionNumber] = useState(1);
+  const [questions, setQuestions] = useState([]);
+  const nextQuestion = (question) => {
+    if (questionNumber < (Number(quiz.noOfQuestionsVal) + 1)) {
+      setQuestions([...questions, question]);
+      publish(`ResetInterval${quiz.quizId}`, quiz.timerVal);
+      setQuestionNumber(questionNumber + 1);
+    }
+  };
+  const shouldShowQuiz = questionNumber < (Number(quiz.noOfQuestionsVal) + 1);
+  return (
+    <div>
+      {shouldShowQuiz && (
+        <TimerContainer
+          questionNum={questionNumber}
+          quiz={quiz}
+        />
+      )}
+      {shouldShowQuiz && (
+        <QuestionAnswer
+          questionNum={questionNumber}
+          nextQuestion={nextQuestion}
+          quiz={quiz}
+        />
+      )}
+      <Score answers={questions} />
+      {!shouldShowQuiz
+      && (
+        <Answers
+          answers={questions}
+          shouldShowCorrectAnswers
+        />
+      )}
+      {!shouldShowQuiz
+      && (
+        <Answers
```

Comment:
What is the purpose of rendering the exact same component twice but with one param set false?

Also, invert the convention. By default the param value should be false. If required, set it true instead of the inverse

Reply:
Done. By default the param value is false.